### PR TITLE
Move lib/filters to app/lib

### PR DESCRIPTION
**Pull request by duduribeiro**
In order to remove the lib folder from autoload, this commit moves the
lib/filters into app/lib

### CODE REVIEW

File: app/lib/filters/provider_required.rb [hunk 1]
```diff
@@ -10,7 +12,7 @@ module ControllerMethods
 
     def provider_required
       if current_account.nil? || !current_account.provider?
```
Comment (Martouta) [inline app/lib/filters/provider_required.rb]:
```suggestion
      if !current_account || current_account.buyer?
```

Comment (Martouta) [reply app/lib/filters/provider_required.rb]: Spoken in private :sunglasses: :+1:

File: app/lib/filters/provider_required.rb [hunk 2]
```diff
@@ -8,13 +10,11 @@ def provider_required(options = {})
...
     protected
 
-    def provider_required
-      if current_account.nil? || !current_account.provider?
-        render_error 'Access denied', :status => :forbidden
-        false
-      else
-        true
-      end
+    def provider_require
```
Comment (Martouta) [inline app/lib/filters/provider_required.rb]:
```suggestion
    def provider_required
```

:see_no_evil:

Comment (duduribeiro) [reply app/lib/filters/provider_required.rb]: 🤦‍♂

Comment (Martouta) [reply app/lib/filters/provider_required.rb]: :joy:

**Pull request merged**